pkgs/tools/misc/jfrog-cli/default.nix

```diff
@@ -1,17 +1,18 @@
 { lib
 , buildGoModule
 , fetchFromGitHub
+, gitUpdater
 }:
 
 buildGoModule rec {
   pname = "jfrog-cli";
-  version = "2.52.10";
+  version = "2.55.0";
 
   src = fetchFromGitHub {
     owner = "jfrog";
     repo = "jfrog-cli";
     rev = "refs/tags/v${version}";
-    hash = "sha256-sqKlYHOpjIxNff1QK540Xxqr7w+WZ+pZXgfAKjRbMuM=";
+    hash = "sha256-GwhEkoPHBsWyEO0H7BU7HM6t8aJ6iNKktkYiyFj9c80=";
   };
 
   vendorHash = "sha256-Cm6Wu2U2i6WZfps1WLAjYuVZZ1y7J4WXFLmX2bkevd4=";
```

Comment:
4 neutral checks, doesn't look good. Does this need to be updated as well?

Reply:
You're right, it does.  I should have used `nix-update-script` for `passthru.updateScript`.  I'm sorry, I don't know how I didn't catch this.